src/lib/components/molecules/table/index.jsx

```diff
@@ -1,10 +1,10 @@
 import { useState } from 'preact/hooks'
 import { useTable } from './useTable'
 import { Chevron } from '$particles/chevron'
-import defaultStyles from './style.module.css'
+import defaultStyles from './style.module.scss'
 import { mergeStyles } from '$styles/helpers/mergeStyles'
 
-export function Table({ columns, data, styles }) {
+export function Table({ columns, data, hideHeader, styles }) {
```

Comment:
Let's give this a default value of `false` to simplify our lives, i.e. `hideHeader = false`.

Reply:
yes sure